source/tutorials/test-strategy/test-strategy.rst

```diff
@@ -54,41 +54,20 @@ Android Validation Out-Scope
 * Pre-Certification
 * Documentation
 
-Validation Activates Key Flow
+Validation Activity Flow
 -----------------------------
```

Comment:
The length of underlying characters must match the length of the section title, otherwise Sphinx will throws "WARNING: Title underline too short" warnings while generating the html files.

Reply:
Got it; will correct.